src/GafferSceneUI/CropWindowTool.cpp

```diff
@@ -459,7 +459,9 @@ class CropWindowTool::Rectangle : public GafferUI::Gadget
 			else
 			{
 				const ImageGadget *imageGadget = static_cast<const ImageGadget *>( viewportGadget->getPrimaryChild() );
-				return imageGadget->pixelAt( event.line );
+				V2f pixel = imageGadget->pixelAt( event.line );
+				pixel.x *= imageGadget->getImage()->format().getPixelAspect();
```

Comment:
I don't _think_ we need to worry about `getImage()` returning `nullptr` do we?

Reply:
ImageGadget appears to have internal support for null images, but I think in practice the ImageGadget in the ImageView will always have an image.